docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md

```diff
@@ -0,0 +1,137 @@
+---
+title: 'setprchsreq.a'
+order: 32
+deploy: ['experimental']
+---
+
+# setprchsreq.a
+
+This action is used to set purchase requirements for a token factory.
+
+## Technical Behavior
+
+Factory manager can specify purchase options for users. Note that currently have to use the same action for both creation and modification of purchase requirements.
```

Comment:
The factory manager can specify purchase options for users. Note that currently they have to use the same action for both creation and modification of purchase requirements.